pex/http.py

```diff
@@ -134,6 +134,8 @@ def resolve(self, link):
 class UrllibContext(Context):
   """Default Python standard library Context."""
 
+  USER_AGENT = 'pex/%s' % PEX_VERSION
+
   def open(self, link):
     return urllib_request.urlopen(link.url)
```

Comment:
This is the other urlopen that needs a UA.

Reply:
Done.